src/rtrc.js

```diff
@@ -219,7 +219,7 @@
 		itemClass = [];
 
 		if (rc.type === 'new') {
-			typeSymbol += '<span class="newpage">N</span>';
+			typeSymbol += '<span class="newpage">' + mw.msg('newpageletter') + '</span>';
```

Comment:
This may need escaping. Use `mw.message().escaped()` for consistency with the rest.